src/community/utrecht/alert.tokens.json

```diff
@@ -0,0 +1,77 @@
+{
+  "utrecht": {
+    "alert": {
+      "icon": {
+        "gap": {
+          "value": "16px"
+        },
+        "error": {
+          "color": {
+            "value": "var(--of-color-danger)"
+          }
+        },
+        "info": {
+          "color": {
+            "value": "var(--of-color-info)"
+          }
+        },
+        "warning": {
+          "color": {
+            "value": "var(--of-color-warning)"
+          }
+        },
+        "ok": {
+          "color": {
+            "value": "var(--of-color-success)"
+          }
+        }
+      },
+      "padding": {
+        "block": {
+          "end": {
+            "value": "20px"
+          },
+          "start": {
+            "value": "20px"
+          }
+        },
+        "inline": {
+          "start": {
+            "value": "20px"
+          },
+          "end": {
+            "value": "20px"
+          }
+        }
+      },
+      "error": {
+        "background": {
+          "color": {
+            "value": "var(--of-alert-error-bg)"
+          }
+        }
+      },
+      "info": {
+        "background": {
+          "color": {
+            "value": "var(--of-alert-info-bg)"
+          }
+        }
+      },
+      "warning": {
+        "background": {
+          "color": {
+            "value": "var(--of-alert-warning-bg)"
+          }
+        }
+      },
+      "ok": {
+        "background": {
+          "color": {
+            "value": "#ddffdd"
```

Comment:
We didn't have a "ok" style, so I took this colour from the backend "color success bg". Can be changed if not liked :smile: